CLAUDE.md

```diff
@@ -0,0 +1,391 @@
+# CLAUDE.md - Claude Code UI Project Guide
+
+## 🚀 Quick Start Commands
+
+### Development
+
+**Local Development:**
+```bash
+# Start development server (frontend + backend)
+npm run dev
+
+# Start backend only
+npm run server
+
+# Start frontend only  
+npm run client
+
+# Build for production
+npm run build
+```
+
+**Docker Development:**
+```bash
+# Start with Docker Compose (recommended)
+docker-compose -f docker-compose.dev.yml up
```

Comment:
`docker-compose` is deprecated in favour of `docker compose`. [Deprecated since July 2023](https://forums.docker.com/t/docker-compose-vs-docker-compose/137884/8)